README.md

```diff
@@ -5,17 +5,25 @@
 
 A simplified API to Intel(R) oneAPI Data Analytics Library that allows for fast usage of the framework suited for Data Scientists or Machine Learning users.  Built to help provide an abstraction to Intel(R) oneAPI Data Analytics Library for either direct usage or integration into one's own framework and extending this beyond by providing drop-in paching for scikit-learn.
 
+Running full scikit-learn test suite with daal4py optimization patches:
+
+- [![CircleCI](https://circleci.com/gh/IntelPython/daal4py.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py) when applied to scikit-learn from PyPi
+- [![CircleCI](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master) when applied to build from master branch
+
+## Follow us on Medium
+- [Improve the Performance of XGBoost and LightGBM Inference](https://medium.com/intel-analytics-software/improving-the-performance-of-xgboost-and-lightgbm-inference-3b542c03447e)
+- [Accelerate Kaggle Challenges Using Intel AI Analytics Toolkit](https://medium.com/intel-analytics-software/accelerate-kaggle-challenges-using-intel-ai-analytics-toolkit-beb148f66d5a)
+- [Accelerate Your scikit-learn Applications](https://medium.com/intel-analytics-software/improving-the-performance-of-xgboost-and-lightgbm-inference-3b542c03447e)
+- [Accelerate Linear Models for Machine Learning](https://medium.com/intel-analytics-software/accelerating-linear-models-for-machine-learning-5a75ff50a0fe)
+- [Accelerate K-Means Clustering](https://medium.com/intel-analytics-software/accelerate-k-means-clustering-6385088788a1)
```

Comment:
Are we going to list all our articles or only latest 5?

I think it worth mentioning that those a blogs  - in subject. It might be not clear for users.

Might be good to add some embedding or pictograms to visualize that this is medium/blogs

Reply:
> Are we going to list all our articles or only latest 5?

That's all I found. When there will be too much, we can put it in a separate section.

> I think it worth mentioning that those a blogs - in subject. It might be not clear for users.

"Follow us on Medium" I think it should be clear that we are talking about articles

> Might be good to add some embedding or pictograms to visualize that this is medium/blogs

Do you know an example?

Reply:
All of these are in this space https://medium.com/intel-analytics-software, right?
I'd say. you can introduce it like this:

```
We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-software) to learn tips and tricks for more efficient data analysis. Here are our latest blogs:
```

Or maybe something else instead of `to learn tips and tricks for more efficient data analysis`, not sure about this phrasing, but you get the idea

Reply:
That's better?